Apromore-Database/src/main/java/org/apromore/dao/model/Usermetadata.java

```diff
@@ -79,6 +79,10 @@
      * Set of UserMetadataLog from linked table
      */
     private Set<UsermetadataLog> usermetadataLogSet = new HashSet<>();
+    /**
+     * Set of UserMetadataLog from linked table
+     */
+    private Set<UsermetadataProcess> usermetadataProcessSet = new HashSet<>();
 
```

Comment:
Can we keep naming consistent across all classes and methods regarding "Usermetadata" or "UserMetadata"?
Some examples: Usermetadata, UsermetadataLog, UsermetadataProcess, UserMetadataBuilder, UsermetadataType, getUserMetadataByLog, getUserMetadata, GroupUsermetadata.

Reply:
It's a compromise due to JPA's naming convention.